pyatlan/version.txt

```diff
@@ -1 +1 @@
-1.3.3
+1.3.4
```

Comment:
Given the new embedded clients are fairly extensive, I'd be tempted to bump this version to 1.4.0 — thoughts?

Reply:
Agreed.